src/main/resources/network/brightspots/rcv/config_file_documentation.txt

```diff
@@ -132,17 +125,15 @@ Config file must be valid JSON format. Examples can be found in the test_data fo
         example: 150
         value: [0..1000000]
 
-      "maxSkippedRanksAllowed" optional
-        maximum number of skipped ranks (undervotes) on a ballot before the ballot should be considered exhausted
+      "maxSkippedRanksAllowed" required
+        maximum number of skipped ranks (undervotes) on a ballot before the ballot should be considered exhausted; if "unlimited" is entered, a ballot will never be considered exhausted
```

Comment:
the wording of "a ballot will never be considered exhausted" is a bit ambiguous. you mean that it will never be considered exhausted _due to skipped ranks_

Reply:
Done.